salt/states/x509.py

```diff
@@ -150,6 +150,33 @@
         - CN: www.example.com
         - days_remaining: 30
         - backup: True
+        - managed_private_key:
```

Comment:
`managed_private_key` was removed on purpose here: https://github.com/saltstack/salt/pull/52935/files#diff-5499a295a50d60a761c34f4080e4014bL80  in favor of using the separate `x509.private_key_managed` state. Can you remove the reference here and on line 175.

Reply:
Hey @Ch3LL, good catch. I remember seeing the change in the code but forgot to look for it in the docs. Just removed it, thanks!